Documentation/DataProcessing/_PageContentFetchingProcessor/_BackendLayout.tsconfig

```diff
@@ -0,0 +1,41 @@
+mod.web_layout.BackendLayouts {
+    default {
```

Comment:
In TYPO3 we have a 2 letter indent: https://github.com/TYPO3/typo3/blob/main/.editorconfig#L51
Or do we need the 4 letter indent for better rst rending?

Reply:
Nope, you can fix the editorconfig in a follow up. I think 2 space indent is better anyway